tools/diff-processor/cmd/changed_schema_resources.go

```diff
@@ -10,51 +10,42 @@ import (
 	"os"
 
 	"github.com/GoogleCloudPlatform/magic-modules/tools/diff-processor/diff"
-	"github.com/GoogleCloudPlatform/magic-modules/tools/issue-labeler/labeler"
 	"github.com/spf13/cobra"
 	"golang.org/x/exp/maps"
 )
 
-const changedSchemaLabelsDesc = `Compute service labels based on schema changes.`
+const changedSchemaResourcesDesc = `Compute service labels based on schema changes.`
```

Comment:
we may want to change the wording since we no longer compute the label?

Reply:
Fixed in 79f8b5e6b